buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -260,7 +260,10 @@ public class PluginBuildPlugin extends BuildPlugin {
         File noticeFile = project.pluginProperties.extension.noticeFile
         if (noticeFile != null) {
             NoticeTask generateNotice = project.tasks.create('generateNotice', NoticeTask.class)
-            generateNotice.dependencies(project)
+            File licensesDir = new File(project.projectDir, 'licenses')
```

Comment:
I don't think you need this. We can default the `licensesDirs` to contain this directory for the project when the task is added?